src/base.jl

```diff
@@ -7,11 +7,25 @@
 # -
 
 
+"""
+    AbstractIVP{T,Y}
+
+A progenitor of types representing an IVP (initial value
```

Comment:
Had to look `progenitor` up, maybe something less fancy? "The abstract supertype of all IVP..."